src/client/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 set(client_SRCS
+	${CMAKE_CURRENT_SOURCE_DIR}/meshgen/collector.cpp
```

Comment:
Also needs to be added to https://github.com/minetest/minetest/blob/master/build/android/jni/Android.mk

Reply:
Done